ddl/index.go

```diff
@@ -352,8 +357,13 @@ func onAlterIndexVisibility(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64,
 	if err != nil || tblInfo == nil {
 		return ver, errors.Trace(err)
 	}
-	idx := tblInfo.FindIndexByName(from.L)
-	idx.Invisible = invisible
+
+	if job.MultiSchemaInfo != nil && job.MultiSchemaInfo.Revertible {
+		job.MarkNonRevertible()
+		return updateVersionAndTableInfo(d, t, job, tblInfo, false)
```

Comment:
Do we need use `updateVersionAndTableInfoWithCheck` instead of `updateVersionAndTableInfo`?

Reply:
The `TableInfo` is not changed. Is it really necessary?

Reply:
It is not necessary